osd/cluster-wide_proxy_ca_expiring.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Info",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Action required: Cluster proxy CA Expiring",
+    "description": "Your cluster requires you to take action because 'Additional Trusted CA Bundle' used for proxy server will not be valid after ${DATE}. Ensure renewing certificate prior to expiration to avoid cluster degredation and/or unavailability.",
```

Comment:
```suggestion
    "description": "Your cluster requires you to take action because one of the Certificate Authority (CA) certificates in the Additional Trusted CA Bundle you have set for your cluster will not be valid after ${DATE}. To avoid any potential disruption of service or cluster availability caused by the expiration of the CA certificate, please ensure your Additional Trusted CA Bundle is updated with any new CA certificates required.",
```